docs/Makefile

```diff
@@ -1,18 +1,21 @@
-PREFIX := /usr/local
-DATADIR := $(PREFIX)/share
-MANDIR := $(DATADIR)/man
+PREFIX ?= /usr/local
+DATADIR ?= $(PREFIX)/share
+MANDIR ?= $(DATADIR)/man
 GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
 
 docs: $(patsubst %.md,%.1,$(wildcard *.md))
 
 %.1: %.md
 	$(GOMD2MAN) -in $^ -out $@
 
-.PHONY: install
-install: docs
+.PHONY: install-nobuild
+install-nobuild:
 	install -d ${MANDIR}/man1
 	install -m 0644 oci-seccomp-bpf-hook*.1 ${MANDIR}/man1
```

Comment:
Not really related to this patch, but looks like `${DESTDIR}` is missing from ^^^ here, and it would also be required by packaging.